LEGO1/mxpresenter.cpp

```diff
@@ -46,52 +49,57 @@ long MxPresenter::StartAction(MxStreamController *, MxDSAction *)
   return 0;
 }
 
-// OFFSET: LEGO1 0x100d4d74 STUB
+// OFFSET: LEGO1 0x100b4e40 STUB
 void MxPresenter::EndAction()
 {
   // TODO
 }
 
-// OFFSET: LEGO1 0x100d4d8c STUB
+// OFFSET: LEGO1 0x100b52d0 STUB
 void MxPresenter::Enable(unsigned char)
 {
   // TODO
 }
 
-// OFFSET: LEGO1 0x1000be30 STUB
+// OFFSET: LEGO1 0x1000be30
 void MxPresenter::VTable0x14()
 {
-  // TODO
 }
 
-// OFFSET: LEGO1 0x1000be40 STUB
+// OFFSET: LEGO1 0x1000be40
 void MxPresenter::VTable0x18()
 {
-  // TODO
+  ParseExtra();
+  m_unk0xc |= 1 << (unsigned char)m_unk0x8;
+  m_unk0x8 = 2;
```

Comment:
From the usage of these two fields it seems they are something like:
(u?)int m_8_tickleState.
DWORD m_C_tickleBits; // bitfield

Also, while I haven't checked with the compiler, does it perhaps do the cast to uchar automatically when given an (u)int? After all, it makes no sense to shift more than 31 bits, and `lsh` is AFAIK also only accepting an 8-bit shift count. Might be worth checking, to possibly get rid of all the manual shiftcount casts.

Reply:
It is also possible the 1 should be unsigned. 1u. I haven't investigated deeply.